transcriptic_zsh_complete.sh

```diff
@@ -0,0 +1,28 @@
+_transcriptic_completion() {
```

Comment:
nit: would prefer this being rolled into another PR but it's fine either way

Reply:
reverting for separate diff :)